src/style.css

```diff
@@ -2,0 +1,69 @@
+body {
+  margin: 0 8px;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
The checklist suggests to use fallback fonts - alternative font-family in case the main one doesn't work. You can add a more common font as a fallback. For example: `font-family: Avenir, Arial, sans-serif;`